elkserver/install-elkserver.sh

```diff
@@ -452,18 +459,18 @@ if [ $ERROR -ne 0 ]; then
     echoerror "Could not set the default index for Kibana (Error Code: $ERROR)."
 fi
 
-echo "Installing GeoIP index template adjustment"
-curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/redirtraffic- -d@./templates/elasticsearch-template-geoip-es7x.json >> $LOGFILE 2>&1
+echo "Installing Elasticsearch ILM policy"
+curl -X PUT "http://localhost:9200/_ilm/policy/redelk" -H "Content-Type: application/json" -d @./templates/redelk_elasticsearch_ilm.ndjson; done >> $LOGFILE 2>&1
 ERROR=$?
 if [ $ERROR -ne 0 ]; then
-    echoerror "Could not install GeoIP index template adjust (Error Code: $ERROR)."
+    echoerror "Could not install Elasticsearch ILM policy (Error Code: $ERROR)."
 fi
 
-echo "Setting elasticsearch to single shards"
-curl -XPUT http://localhost:9200/_template/default -H 'Content-Type: application/json' -d '{"index_patterns": ["*"],"order": -1,"settings": {"number_of_shards": "1","number_of_replicas": "0"}}'  >> $LOGFILE 2>&1
```

Comment:
No longer needed as part of the ES index templates

Reply:
Great addition!